src/Interpreters/AsynchronousMetrics.cpp

```diff
@@ -674,21 +694,32 @@ void AsynchronousMetrics::update(std::chrono::system_clock::time_point update_ti
         {
             Int64 amount = total_memory_tracker.get();
             Int64 peak = total_memory_tracker.getPeak();
-            Int64 new_amount = data.resident;
+            Int64 rss = data.resident;
```

Comment:
Let's also add `amount` and `peek` into metrics?

```suggestion
            new_values["MemoryTrackerAmount"] = amount;
            new_values["MemoryTrackerPeak"] = peak;

            Int64 rss = data.resident;            
```

Reply:
"MemoryTrackerAmount" is "MemoryTracking" afaik, peak sounds reasonable

Reply:
But you removed `CurrentMetrics::set(CurrentMetrics::MemoryTracking, new_amount)`, so there is no value of `total_memory_tracker.get()` in metrics in this patch. Am I missing something?